tests/test_acasclient.py

```diff
@@ -22,6 +21,8 @@
 M  END
 """
 
+ACAS_NODEAPI_BASE_URL = "http://localhost:3001"
```

Comment:
Trying to think of a way to communicate a message when a someone doesn't have access to port 3001.  Wonder if a decorator on these routes that use this would work.

Reply:
Good suggestion. I've implemented this as a decorator and done some testing by temporarily tweaking the ACAS_NODEAPI_BASE_URL to 30001 to simulate inaccessibility. As far as I can tell, the tests will run without port 3001 as long as bob and Global have already been created, and otherwise they will all be skipped. And my last test which explicitly requires the node api to create a test user will always be skipped unless node api is available.

Reply:
Nice!